src/librbd/Types.h

```diff
@@ -89,9 +89,10 @@ struct SnapInfo {
 };
 
 enum {
-  OPEN_FLAG_SKIP_OPEN_PARENT = 1 << 0,
-  OPEN_FLAG_OLD_FORMAT       = 1 << 1,
-  OPEN_FLAG_IGNORE_MIGRATING = 1 << 2
+  OPEN_FLAG_SKIP_OPEN_PARENT       = 1 << 0,
+  OPEN_FLAG_OLD_FORMAT             = 1 << 1,
+  OPEN_FLAG_IGNORE_MIGRATING       = 1 << 2,
+  OPEN_FLAG_IGNORE_DATA_POOL_ERROR = 1 << 3,
```

Comment:
Nit: should this be more specific? i.e. `OPEN_FLAG_IGNORE_DATA_POOL_MISSING` since if you have a permission error we wouldn't want to skip trimming the image during a remove.